src/configs/starship_root.rs

```diff
@@ -56,6 +56,7 @@ pub const PROMPT_ORDER: &[&str] = &[
     "scala",
     "swift",
     "terraform",
+    "v",
```

Comment:
If `go` is marked as `golang` this should be `vlang` instead.
```suggestion
    "vlang",
```

Reply:
Okay, let me change that.